File: 2.Implement/Delivery_Server/src/main/java/com/delivery/app/Delivery/data/dto/response/cart/SaveCartResponseDto.java
package com.delivery.app.Delivery.data.dto.response.cart;

import com.delivery.app.Delivery.data.dto.response.BaseResponseDto;

public class SaveCartResponseDto implements BaseResponseDto{
    private Long id;

    public SaveCartResponseDto(Long id) {
        this.id = id;
    }
}

File: 2.Implement/DeliveryApp/app/src/main/java/com/project/deliveryapp/retrofit/dto/response/cart/SaveCartResponseDto.kt
package com.project.deliveryapp.retrofit.dto.response.cart

import com.project.deliveryapp.retrofit.dto.ResponseDto
import org.json.JSONObject

class SaveCartResponseDto(json: JSONObject): ResponseDto {
    val id: Long

    init {
        val obj = json.getJSONObject("responseDto")

        id = obj.getLong("id")
    }
}